server/src/main/java/org/elasticsearch/cluster/metadata/DataStream.java

```diff
@@ -156,8 +153,7 @@ public DataStream(
         );
     }
 
-    // visible for testing
-    DataStream(
+    public DataStream(
```

Comment:
Do we now use this constructor in more places than before? Are those places outside of the current package? Should we instead be using the builder interfaces?

Reply:
Ah no this visibility change was by accident.

Reply:
Removed the constructor altogether as per one of the above discussions :).